update MY Apps URL to point to Sequent developer - My Apps portal ins… (#58)
SeequentEvo/evo-mcp: README.md

Make a copy of the file `.env.example` and rename it to `.env`. Fill in your app credentials as described below.

##### Evo app credentials

> You first need to create a **native app** in the [Seequent Evo My Apps page](https://developer.seequent.com/my-apps). This app will allow you to sign in with your Bentley account to access Seequent Evo. Visit the [Evo Developer Portal](https://developer.seequent.com/docs/guides/getting-started/apps-and-tokens) to learn more.

Reviewer: ```suggestion
You first need to create a **native app** in the [My apps](https://developer.seequent.com/my-apps) page of the [Seequent Developer Portal]. This app will allow you to sign in with your Bentley account to access Seequent Evo. Visit the [Seequent Developer Portal](https://developer.seequent.com/docs/guides/getting-started/apps-and-tokens) to learn more.
```

Author: implemented the changes in new commit , please verify

<details>
<summary><strong>Alternative: Service authentication (for automation/CI)</strong></summary>

> If you need to run the server without interactive sign-in (e.g. automation, CI/CD, or background services), you can use a **service app** instead. Create a service app in the [Seequent Evo My Apps page](https://developer.seequent.com/my-apps) Developer Portal and set the following in your `.env` file:

Reviewer: ```suggestion
If you need to run the server without interactive sign-in (e.g. automation, CI/CD, or background services), you can use a **service app** instead. Create a **service app** in the [My apps](https://developer.seequent.com/my-apps) page of the Seequent Developer Portal and set the following in your `.env` file:
```

Author: implemented the changes in new commit , please verify

**Redirect URL configuration:**

>   Based on the authentication mode, the redirect URL for OAuth will be configured differently. Ensure you set the correct redirect URL in My Apps in app registration and the correct environment variable in your `.env` file.

Reviewer: ```suggestion
  Based on the authentication mode, the redirect URL for OAuth will be configured differently. Ensure you set the correct redirect URL in your [app configuration](https://developer.seequent.com/my-apps) and the correct environment variable in your `.env` file.
```

Author: implemented the changes in new commit , please verify
